src/actions/account.js

```diff
@@ -323,8 +354,13 @@ export const { addAccessKey, createAccountWithSeedPhrase, addAccessKeySeedPhrase
             const publicKey = recoveryKeyPair.publicKey.toString()
             const contractName = null;
             const fullAccess = true;
-            await wallet.addAccessKey(accountId, contractName, publicKey, fullAccess)
-            await wallet.postSignedJson('/account/seedPhraseAdded', { accountId, publicKey })
+            
+            try {
+                await wallet.addAccessKey(accountId, contractName, publicKey, fullAccess)
+                await wallet.postSignedJson('/account/seedPhraseAdded', { accountId, publicKey })
```

Comment:
Shoudln't `/account/seedPhraseAdded` happen before `addAccessKey` here (same as above)?

Reply:
It's intentional. Because there is a scenario where the user might end up with two seed phrase recovery:
 - CH added a recovery option
 - `addAccessKey` throw error but the key will be added in few minutes
 - in the meantime, user tries to add seed phrase recovery the second time and succeded
 - user ends with two seed phrase recovery options.

I have an idea to make it work a bit better. In this case:
 - CH added a recovery option
 - `addAccessKey` throw error but the key will be added in few minutes
 -  user is redirected to `/profile` but we are showing that the recovery option is `pending` and two options:
   - `refresh` button so he can check if the key was added by `addAccessKey`
   - `cancel` button - so he can clear the key from the chain (if exists) and CH and try again.

What do you think about that? It's a bit more complicated than the implemented solution, but in this case, the user might try to rescue the seed phrase that he already keeps somewhere.

Reply:
> Because there is a scenario where the user might end up with two seed phrase recovery

not sure how current code solves this. With current code:
- `addAcessKey` can succeed, but `/seedPhraseAdded` fail
- in this case user in reality has 2 seed phrases, but only one shows up in UI

it is far better to end up with 2 seed phrases which show up in UI, than with broken state where there is some key for which we don't know where it belongs

@abacabadabacaba wonder whether we can have any good way for user to identify seed phrases without compromising security. E.g. can storing either first or last word compromise security?  Should we generate one extra word just for identification?

Reply:
When I was discussing changes on Slack, Matt pointed out that "I think we don't want to focus on the CH case though since that will likely never be happening on mainnet.". 
Is this accurate?

Reply:
> Matt pointed out that "I think we don't want to focus on the CH case though since that will likely never be happening on mainnet.".

not sure what do you mean here? If the assumption that `/seedPhraseAdded` won't every be happening on mainnet – I don't think this is good assumption at all. It is happening I'm pretty sure. Every HTTP request can fail intermittently even if everything is running ok (e.g. network connection issue).